test/test-install.js

```diff
@@ -44,3 +52,119 @@ test('EACCES retry once', async (t) => {
     }
   }
 })
+
+// only run these tests if we are running a version of Node with predictable version path behavior
+const skipParallelInstallTests = process.env.FAST_TEST ||
+  process.release.name !== 'node' ||
+  semver.prerelease(process.version) !== null ||
+  semver.satisfies(process.version, '<10')
+
+test('parallel installs (ensure=true)', async (t) => {
```

Comment:
Nitpicking: All `parallel installs ...` tests are similar (only differ in `prog.opts`). Potentially they can be refactored to a parametrized function to avoid code duplication.

Reply:
Done.